.github/workflows/release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Run GoReleaser
       uses: goreleaser/goreleaser-action@v2
       with:
-        version: v1.18.2
```

Comment:
im not too sure about this being ideal, I opened https://github.com/shipwright-io/cli/pull/227 so dependabot can take care of it.

Reply:
I think this will not be tackled by the Dependabot automation. I would actually suggest to remove the version here as we don't want to pin the GoReleaser version unless we have a very good reason.

Reply:
But we have a good reason to pin the GoReleaser version: https://github.com/shipwright-io/cli/pull/204. We should have maybe created an issue at that time, though.

Reply:
I wasn't aware, or I forgot about it. But I would suggest we just fix the issue: https://github.com/shipwright-io/cli/pull/233 to get out of that pinning.

Reply:
@Adarsh-jaiss Now that we have all understood the implications and with #233 being merged, can you rebase your PR one more time? The change of the version here is no longer needed. Thanks!